rcl_action/QUALITY_DECLARATION.md

```diff
@@ -2,9 +2,9 @@ This document is a declaration of software quality for the `rcl_action` package,
 
 # `rcl_action` Quality Declaration
 
-The package `rcl_action` claims to be in the **Quality Level 4** category.
+The package `rcl_action` claims to be in the **Quality Level 2** category.
```

Comment:
`rcl_action` can't be level 2 yet because it depends on `action_msgs`

Reply:
That would also be the case for rcl_lifecycle then too, I think.